### ISSUE TITLE
UBFC-rPPG data set segmentation

### ISSUE THREAD

**Issue by JasonYpro** [zizheng-guo/RhythmMamba#3]
Hi, thank you very much for your excellent work. I try to verify with RhythmMamba on UBFC-rppg. At first, I follow the default configuration and do not use verification set, that is, 0-0.72 is the training set and 0.72-1 is the test set. The data partition display is correct, but why is it shown that only 22 videos have been read during training? Shouldn't it be 30 videos? The test set is 12 and that's right. I later tried using 0.72-1 as the validation set and found that the training set read 22 videos, the validation set read 9 videos, and the test set read 12 videos. Is this correct?
![image](https://github.com/user-attachments/assets/df507343-f436-4dd9-839b-db904519e8b9)

**Comment by JasonYpro** [comment]
The data preprocessing process should have been correct, but only 22 videos were read during training
![image](https://github.com/user-attachments/assets/89e72e51-3eb4-46c9-8841-0e606d730cee)

**Comment by zizheng-guo** [comment]
The protocol used in the paper is that the first 30 videos are the training set, the last 12 are the test set, and no verification set is used.
You can view the partitioning during preprocessing in detail in the split_raw_data function of UBFCrPPGLoader.py.

Your preprocessing output seems to be fine, the 22 shown during training is not the meaning of 22 videos, but an epoch consists of 22 batches, the number of batches depends on the batchsize you set and the length of the video partition (CHUNK_LENGTH).
